resources/sass/global/_navbar.scss

```diff
@@ -0,0 +1,32 @@
+@import './variables';
+
+.navbar{
+    background-color: $i-light-blue;
+    color: $i-white;
+    position: absolute;
+    top: 0;
+    z-index: 1;
+    width: 100%;
+    &__dropdown{
+        height: 100vh;
```

Comment:
When the *navbar* is expanded, `.navbar__dropdown` is extending past the screen because its starting after the *navbar button*. You could possibly fix this by setting height at `.navbar` but will require addition css fixes to adjust the content.

Reply:
i gotchu!